gradio/components/html.py

```diff
@@ -45,11 +67,16 @@ def __init__(
         container: bool = False,
         padding: bool = False,
         autoscroll: bool = False,
+        **props: dict[str, Any],
     ):
         """
         Parameters:
-            value: The HTML content to display. Only static HTML is rendered (e.g. no JavaScript. To render JavaScript, use the `js` or `head` parameters in the `Blocks` constructor). If a function is provided, the function will be called each time the app loads to set the initial value of this component.
+            value: The HTML content in the ${value} tag in the html_template. For example, if html_template="<p>${value}</p>" and value="Hello, world!", the component will render as `"<p>Hello, world!</p>"`.
             label: The label for this component. Is used as the header if there are a table of examples for this component. If None and used in a `gr.Interface`, the label will be the name of the parameter this component is assigned to.
+            html_template: A string representing the HTML template for this component as a JS template string and Nunjucks template. The `${value}` tag will be replaced with the `value` parameter, and all other tags will be filled in with the values from `props`.
+            css_template: A string representing the CSS template for this component as a JS template string and Nunjucks template. The CSS will be automatically scoped to this component. The `${value}` tag will be replaced with the `value` parameter, and all other tags will be filled in with the values from `props`.
+            js_on_load: A string representing the JavaScript code that will be executed when the component is loaded. The `element` variable refers to the HTML element of this component, and can be used to access children such as `element.querySelector()`. The `trigger` function can be used to trigger events, such as `trigger('click')`. The value and other props can be edited through `props`, e.g. `props.value = "new value"` which will re-render the HTML template.
+            apply_default_css: If True, default Gradio CSS styles will be applied to the HTML component.
```

Comment:
Just to confirm, if `False`, no default styling is applied whatsover?

Reply:
We should mention this in the guide, and perhaps show an example with `apply_default_css=False` and how to handle light/dark mode manually